src/sass/_Fabric.Typography.scss

```diff
@@ -344,8 +344,12 @@
   color: $ms-color-success;
 }
 
-@mixin ms-fontColor-alert {
-  color: $ms-color-alert;
+@mixin ms-fontColor-warning {
```

Comment:
This is a breaking change. For now, can you add `ms-fontColor-warning` and mark `ms-fontColor-alert` and its variables as deprecated without removing them?